tests/python_tests/test_vlm_pipeline.py

```diff
@@ -442,21 +441,30 @@ def test_perf_metrics(cache, backend):
 
 @pytest.mark.precommit
 @pytest.mark.nightly
-# FIXME: katuni4ka/tiny-random-qwen2vl and katuni4ka/tiny-random-qwen2.5-vl - fails on NPU
-@pytest.mark.parametrize("model_id", model_ids[:-2])
+@pytest.mark.parametrize("model_id", model_ids)
 @pytest.mark.parametrize("backend", attention_backend)
 @pytest.mark.skipif(
     sys.platform == "darwin" or platform.machine() in ["aarch64", "arm64", "ARM64"],
     reason="NPU plugin is available only on Linux and Windows x86_64",
 )
 def test_vlm_npu_no_exception(model_id, backend):
-    models_path = get_ov_model(model_ids[0])
+    unsupported_models = model_ids = {
+        "katuni4ka/tiny-random-internvl2",
+    }
```

Comment:
`model_ids` is overridden. Is it intentional?

Reply:
No, thanks!